src/EnergyPlus/ExhaustAirSystemManager.hh

```diff
@@ -0,0 +1,180 @@
+//     reference solely to the software portion of its product, Licensee must refer to the
+//     software as "EnergyPlus version X" software, where "X" is the version number Licensee
+//     obtained under this License and may not use a different name for the software. Except as
+//     specifically required in this Section (4), Licensee shall not use in a company name, a
+//     product name, in advertising, publicity, or other promotional activities any name, trade
+//     name, trademark, logo, or other designation of "EnergyPlus", "E+", "e+" or confusingly
+//     similar designation, without the U.S. Department of Energy's prior written consent.
+//
+// THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS" AND ANY EXPRESS OR
+// IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE IMPLIED WARRANTIES OF MERCHANTABILITY
+// AND FITNESS FOR A PARTICULAR PURPOSE ARE DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT OWNER OR
+// CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR
+// CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR
+// SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY
+// THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR
+// OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE
+// POSSIBILITY OF SUCH DAMAGE.
+
+#ifndef ExhaustAirSystemManager_hh_INCLUDED
+#define ExhaustAirSystemManager_hh_INCLUDED
+
+// EnergyPlus Headers
+#include <EnergyPlus/Data/BaseData.hh>
+#include <EnergyPlus/EnergyPlus.hh>
+
+namespace EnergyPlus {
+
+// Forward declarations
+struct EnergyPlusData;
+
+namespace ExhaustAirSystemManager {
+
+    struct ExhaustAir
+    {
+        // Members
+        std::string Name;
+
+        int AvailScheduleNum;
+        std::string ZoneMixerName;
+        int ZoneMixerIndex;
+        int CentralFanTypeNum;
+        std::string CentralFanName;
+        int CentralFanIndex;
+
+        bool SizingFlag;
+
+        // output variables
+        Real64 centralFan_MassFlowRate;
+        Real64 centralFan_VolumeFlowRate_Std;
+        Real64 centralFan_VolumeFlowRate_Cur;
+        Real64 centralFan_Power;
+        Real64 centralFan_Energy;
+
+        // Output acc variable for heat rejection outputs
+        Real64 exhTotalHVACReliefHeatLoss; // feed to state.dataHeatBal->SysTotalHVACReliefHeatLoss
+
+        // Default Constructor
+        ExhaustAir()
+            : AvailScheduleNum(0), ZoneMixerName(""), ZoneMixerIndex(0), CentralFanTypeNum(0), CentralFanName(""), CentralFanIndex(0),
+              SizingFlag(true), centralFan_MassFlowRate(0.0), centralFan_VolumeFlowRate_Std(0.0), centralFan_VolumeFlowRate_Cur(0.0),
+              centralFan_Power(0.0), centralFan_Energy(0.0), exhTotalHVACReliefHeatLoss(0.0)
+        {
+        }
```

Comment:
Next time you are creating new classes, just initialize at the variable declaration rather than creating a constructor.  No worries for this PR, but just note it for the future.

Reply:
Thanks! Will keep in mind. I've pushed in the changes with idd change (min-fields added for AirLoopHVAC:ExhaustSystem).